x-pack/test/ui_capabilities/common/services/ui_capabilities.ts

```diff
@@ -42,21 +43,23 @@ export class UICapabilitiesService {
 
   public async get({
     credentials,
-    navLinks,
     spaceId,
   }: {
     credentials?: BasicCredentials;
-    navLinks?: Record<string, boolean>;
     spaceId?: string;
   }): Promise<GetUICapabilitiesResult> {
     const spaceUrlPrefix = spaceId ? `/s/${spaceId}` : '';
-    this.log.debug(`requesting ${spaceUrlPrefix}/api/capabilities to get the uiCapabilities`);
-    const requestOptions = credentials ? { auth: credentials } : {};
-    const response = await this.axios.post(
-      `${spaceUrlPrefix}/api/capabilities`,
-      { capabilities: { navLinks } },
-      requestOptions
-    );
+    this.log.debug(`requesting ${spaceUrlPrefix}/app/kibana to parse the uiCapabilities`);
+    const requestHeaders = credentials
+      ? {
+          Authorization: `Basic ${Buffer.from(
```

Comment:
If this is UI code, I would highly recommend not using Buffer, as it can include a lot of polyfills. Instead, you can use the native, poorly-named, base64 functions:

```js
Authorization: `Basic ${btoa(`${credentials.username}:${credentials.password}`)}`,
```

Reply:
This is test code and will be reverted once #36319 is unblocked.